.github/workflows/pytest.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: pytest
+
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  pytest:
+    runs-on: ubuntu-latest
+
+    timeout-minutes: 15
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
```

Comment:
`setup.cfg` specifies a minimum version of 3.8, so perhaps that should be specified here.

Reply:
Done with 8608a56.